test/new_test.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
```

Comment:
This should have a different name than `new_test.py`. It won't be "new" for long, and then it won't be easy to guess what it tests without looking inside.

Reply:
More importantly, the test isn't actually running on the CI at the moment - I don't see them in the logs here https://github.com/SMTG-UCL/galore/pull/41/checks

That's because the test framework expects files containing tests to begin with the word "test" - this needs to be renamed to something like "test_csv_data.py" so they are found by `setup.py test`.